SQL/database_changelog.txt

```diff
@@ -9,6 +9,15 @@ INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (7, 2);
 In any query remember to add a prefix to the table names if you use one.
 
 
+-----------------------------------------------------
+
+-----------------------------------------------------
+
+Version 7.3, 15 September 2024, by Rkz
+Set default client fps 60
+
+UPDATE SS13_preferences SET "clientfps" = 60
```

Comment:
This is incorrect and doesn't run.

Reply:
This just isn't necessary, since the default value in the database is "-1", which corresponds to the default as configured in the codebase (currently 40, now 60)

We could, however, change any manually configured 40s to 'default' 60s.

```suggestion
UPDATE `SS13_preferences` SET `preference_value` = "-1" WHERE `preference_tag` = "clientfps" AND `preference_value` = "40"
```